app/javascript/tests/subject/MoveToHousehold.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { shallow } from 'enzyme';
+import { Button, Modal, Form } from 'react-bootstrap';
+import ReactTooltip from 'react-tooltip';
+import MoveToHousehold from '../../components/subject/MoveToHousehold.js'
+import { mockPatient1 } from '../mocks/mockPatients'
+
+const authyToken = "Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==";
+
+function getWrapper(patient) {
+    return shallow(<MoveToHousehold patient={patient} authenticity_token={authyToken} />);
+}
+
+function getPatientName(patient) {
+  return `${patient.first_name} ${patient.middle_name} ${patient.last_name}`;
+}
+
+describe('MoveToHousehold', () => {
+    it('Properly renders Move To Household button', () => {
+        const wrapper = getWrapper(mockPatient1);
+        expect(wrapper.find(Button).length).toEqual(1);
+        expect(wrapper.find(Button).text().includes('Move To Household')).toBeTruthy();
+        expect(wrapper.find('i').hasClass('fa-house-user')).toBeTruthy();
+        expect(wrapper.find(Button).prop('disabled')).toBeFalsy();
+        expect(wrapper.find(ReactTooltip).exists()).toBeFalsy();
+    });
+
+    it('Clicking the Move to Household button opens modal and modal renders correctly', () => {
+      const wrapper = getWrapper(mockPatient1);
+      const toggleModalSpy = jest.spyOn(wrapper.instance(), "toggleModal");
+      const updateTableSpy = jest.spyOn(wrapper.instance(), "updateTable");
+      // Necessary for the spy to be called on click
+      wrapper.instance().forceUpdate();
```

Comment:
this shouldn't be necessary - was it not working without it? tends to be an indicator that something else might be up

Reply:
Yes, it would fail otherwise and I read a fair amount about why in threads like [this one](https://github.com/enzymejs/enzyme/issues/944). I did see it seemed to work okay in other instances in our frontend tests, though they weren't _exactly_ the same, but it still seems like a known issue. Another way to resolve it would be to change this `onClick` to be `onClick={() => this.toggleModal()}` rather than `onClick={this.toggleModal}` which is a worse solution. I was actually curious if you had ran into anything like this yourself.

Reply:
I definitely encountered this in certain instances for not others but wasn't able to figure out what the difference was with the different methods (they were all using inline binding). Typically what I do as a work around is test the functionality of the method being called, i.e. that the state updated, etc